docs/devguide/internet_enabled_rtps.rst

```diff
@@ -240,6 +243,10 @@ The command-line options for the RtpsRelay:
 
   Enable/disable logging of activity events.
 
+.. option:: -LogHttp 0|1
+
+  Enable/disable logging of the HTTP server.
```

Comment:
```suggestion
  Enable/disable logging in the HTTP server.
```

?

Does this need to be more verbose to describe (or link to) what the HTTP server is/does?

Reply:
Yes, I can link the options related to the meta discovery and the section describing the endpoints.